stdlib/gc.ml

```diff
@@ -38,6 +38,7 @@ type control = {
   mutable max_overhead : int;
   mutable stack_limit : int;
   mutable allocation_policy : int;
+  window_size : int;
```

Comment:
Done!  One can now write:

````ocaml
type control = {
   mutable minor_heap_size : int
       [@@mutable_deprecated "Use {ctrl with minor_heap_size = ...} instead."];
   ...
````

I didn't actually do the change in Gc for now.